### DISCUSSION TITLE
GitHub Copilot could not connect to server

### DISCUSSION THREAD

**Discussion by kshnkvn** [community/community#47590] [Copilot Conversations] [answered]
I had Copilot installed on MacOS and everything worked fine. I needed to install it in Windows, after that every time I start VSCode I get an error:
```
[INFO] [auth] [2023-02-15T20:02:08.109Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-02-15T20:02:08.112Z] GitHub Copilot could not connect to server. Extension activation failed: "Contact Support"
```
The only way to fix it is to log out of my githab account and log back in. And this has to be done every time I start VSCode on both Windows and MacOS. I've tried reinstalling both VSCode and the plugin.

**Comment by kshnkvn** [comment]
I did logout and deleted on Windows, reinstalled the plugin and re-logged on MacOS - still doesn't work.

**Comment by diminDDL** [comment]
Same issue here, it used to work fine and now I get:
```
[INFO] [auth] [2023-02-16T13:39:19.656Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-02-16T13:39:19.662Z] GitHub Copilot could not connect to server. Extension activation failed: "Contact Support"
```
I have a verified student account so copilot should work.

**Comment by tuves** [comment, marked as answer]
Thank you for reporting this. The bug fix has now been deployed to production and affected users should be able to connect. 

Due to the recency of this incident, we are still investigating the contributing factors and will provide a more detailed update on cause and remediation in the February Availability Report, which we will publish the first Wednesday of March.

We sincerely apologize for any stress and/or frustration this has caused.

**Reply by snowluminum** [reply]
as for 17.02.2023 14:40 still 403

**Reply by Firaenix** [reply]
Not working for me on my mac and windows machines. getting 403 from server

**Comment by setter43** [comment]
I have signed up for a yearly plan and receive the same error msg
`GitHub Copilot could not connect to server. Extension activation failed`
I was a user previously but have only just re-subscribed.

**Comment by Firaenix** [comment]
Same issue, getting 403, if I try to click on the copilot button in my profile to pay, it just redirects back to my profile page.

**Reply by Akash1134** [reply]
Hi @Firaenix ,

Due to current trade controls, your GitHub account might be restricted from making use of Copilot. This is a region-based restriction and applies to all users located in some trade-restricted countries, being flagged for the same earlier reasons, and is separate from other GitHub services. Read more about it here:

[GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls)

If you feel this should not be the case, feel free to contact our Support team and [open a ticket ](https://support.github.com/request?tags=dotcom-footer)with them.

Thanks!

**Reply by Firaenix** [reply]
Yeah I skimmed the restrictions before but I live in Australia and haven't VPN'd or travelled to any of the highlighted restricted countries. Is it possible that the geolocation database needs updating?

Update: it's definitely account related and not region related. If I sign in with a fresh GitHub account, I can go through the Copilot signup and payment steps no problem. I have just disabled my Copilot subscription because I was about to get billed again even though I have been unable to use it for the last half a month or more
